libdirections-offboard/src/main/java/com/mapbox/navigation/route/offboard/router/MapboxDirectionsUtils.kt

```diff
@@ -0,0 +1,226 @@
+package com.mapbox.navigation.route.offboard.router
+
+import com.mapbox.api.directions.v5.MapboxDirections
+import com.mapbox.api.directions.v5.models.RouteOptions
+import com.mapbox.geojson.Point
+import com.mapbox.navigation.base.route.RouteUrl
+import java.util.*
+
+
+private val EVENT_LISTENER = NavigationRouteEventListener()
+private const val SEMICOLON = ";"
+private const val COMMA = ","
+
+fun MapboxDirections.Builder.routeOptions(options: RouteOptions): MapboxDirections.Builder {
+    baseUrl(options.baseUrl())
+    user(options.user())
+    profile(options.profile())
+
+    options.coordinates().forEachIndexed { index, point ->
```

Comment:
We should assert that there are at least 2 points in the array.

Reply:
done